src/custom/components/Web3Status/index.tsx

```diff
@@ -5,27 +5,28 @@ import { NetworkContextName } from 'constants/index'
 
 import WalletModal from 'components/WalletModal'
 import { Web3StatusInner } from './Web3StatusMod'
-import useAllTransactionsAndOrders from 'custom/hooks/useAllTransactionsAndOrders'
+import useRecentActivity from '@src/custom/hooks/useRecentActivity'
```

Comment:
```suggestion
import useRecentActivity from 'hooks/useRecentActivity'
```

Reply:
oooh auto pathing fail VS. when i changed the name i guess, good catch